R/output_table.R

```diff
@@ -12,13 +12,15 @@ output_table <- function(data, model) {
                        customerid = as.character(model$predicted_values$customerid), 
                        stringsAsFactors = FALSE)
   
+  response <- model$model_hyperparameters$dependent_variable
```

Comment:
Does this work in the unsupervised case where there is no dependent_variable and response would most likely be NULL?